rollup.config.js

```diff
@@ -36,6 +37,9 @@ const appConfiguration = {
       tsconfig: "src/app/tsconfig.json",
       sourceMap: watchMode === true,
     }),
+    json({
+      compact: true,
+    }),
```

Comment:
Interesting! What made you try this plugin?

Reply:
The parser requires it to work

Reply:
Fair enough haha!